xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -670,9 +693,22 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that iterates through the collection.</summary>
+        <returns>An enumerator that can be used to iterate through the collection.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The returned <xref:System.Collections.Generic.IEnumerator%601> provides the ability to iterate through the collection by exposing a <xref:System.Collections.Generic.IEnumerator%601.Current%2A> property .You can use enumerators to read the data in a collection, but not to modify the collection.   
```

Comment:
Do we really put this for all GetEnumerator methods?

Reply:
I don't think it hurts to have it, I think it should stay since it's an interesting description for the enumerator. And since this text is in the remarks, it won't show up in Intellisense.